src/components/course/EnrollModal.jsx

```diff
@@ -23,12 +46,37 @@ export const MODAL_TEXTS = {
   HAS_ENTERPRISE_OFFER: {
     body: (offer, courseRunPrice) => createUseEnterpriseOfferText(offer, courseRunPrice),
     button: 'Enroll',
-    title: 'Use learner credit for this course?',
+    title: 'Use learner credit for this course?', // refers to *legacy* learner credit
+  },
+  HAS_LEARNER_CREDIT: {
+    Body: () => (
+      <>
+        {/* TODO: i18n */}
+        <p>This course is covered by your organization, which allows you to upgrade for free.</p>
+        <p>By upgrading, you will get:</p>
+        <ul className="list-unstyled">
+          <Stack gap={1}>
+            <UpgradeConfirmationModalListItem>
+              Unlimited access to course materials
+            </UpgradeConfirmationModalListItem>
+            <UpgradeConfirmationModalListItem>
+              Feedback and graded assignments
+            </UpgradeConfirmationModalListItem>
+            <UpgradeConfirmationModalListItem>
+              Shareable certifcate upon completion
+            </UpgradeConfirmationModalListItem>
+          </Stack>
```

Comment:
[nit] Could use with some DRYing up

```jsx

const modalListItems = [
  "Unlimited access to course materials",
  "Feedback and graded assignments",
  "Shareable certif[i]cate upon completion"
  ]

return (
<Stack gap={1}>
  {modalListItems.map((listItem) => (
      <UpgradeConfirmationModalListItem key={uuidv4()}>{listItem}</UpgradeConfirmationModalListItem>
))}
</Stack>
)
```

Reply:
Also, good opportunity for using `defineMessages` here within the list to iterate over and provide a translated experience

UPDATE/EDIT: Just noticed your TODO 👍🏽  So this bit can be deferred